web/cobrands/tfl/assets.js

```diff
@@ -271,6 +271,22 @@ function is_tlrn_category_only(category, bodies) {
         bodies.length <= 1;
 }
 
+var a13dbfo_categories = [
```

Comment:
This is not the list of categories, which looks to be all of TLRN with a few differences - think would be happy just to go with TLRN for now given the time and it would catch basically everything.